TEMPLATE.md

```diff
@@ -16,21 +16,25 @@
 
 (Description of your use case here, you can use the following subdivision, but you may also deviate from it and use whatever you see fit)
 
-### Data
+### What is the research about?
 
-(if known, describe the data that is needed for the intended research, bne as specific as possible)
+(mention the research project, the research domain, e.g., linguistics or media studies, and research question if relevant)
 
-### What is the problem?
+### What is needed to do the research?
```

Comment:
This invites a list of solutions. I think it would be good to make sure that we stick to problem descriptions here. So perhaps: 

What is currently too hard to achieve that inhibits the research?

Reply:
Good point, the change from describing the problem to proposing solutions is a different perspective. I think both are actually relevant, so it may be good to keep the distinction and keep both items. I have done a commit that address this now (70913a0918685724d2c2d997c317bfd773841130)